Test Harness/NRTestApp/upload_dsym_results.log

```diff
@@ -0,0 +1,2 @@
+New Relic: Starting dSYM upload script...
```

Comment:
This shouldn't be committed and upload_dsym_results.log should be added to .gitignore